src/index.ts

```diff
@@ -59,6 +60,13 @@ import { OrderBook } from "./orderBook";
 })();
 
 async function startApp(whoAmI: string, index: number) {
+    if (!fs.existsSync("./.env")) {
```

Comment:
Good enough, I would have created a function that is ran at the beginning of the main instead.

Reply:
Hmmm the main doesn't use the envfile though

Reply:
The whole app is using the `.env`...